.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,120 @@
+# Tupaia contributions guide
+
+Thank you for taking the time to contribute, and welcome to the Tupaia!
+We value your expertise, insights, and commitment to making Tupaia a global good. This
+document is a guide on how to contribute to Tupaia effectively, ensuring it remains a
+cohesive, maintainable, and globally valuable product.
+
+## Code of conduct
+
+This project and everyone participating in it is governed by the
+[BES Open Source Code of Conduct](code-of-conduct.md).
+By participating, you are expected to uphold this code. Please report unacceptable behaviour to [opensource@bes.au](opensource@bes.au).
+
+## Copyright Attribution
+
+All open-source contributors to Tupaia will be required to sign a Contributor License Agreement (CLA)
+before BES can accept your contribution into the project code. This is done so that we can continue
+to maintain the codebase covered by a single license. Signing a CLA can be done through GitHub
+before merging in your first pull request.
```

Comment:
It really can? That's cool.

Reply:
I actually copied this from Michelle's and it was reviewed by Felix, so I assume that's right? But also happy to remove